### PR TITLE
Update api-files-gatsby-browser.md

### PR DESCRIPTION
**Pull request by spielers**
ill take the smaller files and finish asap let's focus on the big files @visshaljagtap  you take few files. I am also planning to take a few.

### CODE REVIEW

File: docs/docs/api-files-gatsby-browser.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
 title: The gatsby-browser.js API file
 ---
 
-The file `gatsby-browser.js` lets you respond to actions within the browser, and wrap your site in additional components. The [Gatsby Browser API](/docs/browser-apis) gives you many options for interacting with the [client-side](/docs/glossary#client-side) of Gatsby.
+`Gatsby-browser.js` फाइल आपल्याला ब्राउझरमधील क्रियांना प्रतिसाद देऊ देते आणि अतिरिक्त घटकांमध्ये आपली साइट वापरु देते. The [[Gatsby ब्राउझर API](/docs/browser-apis)आपल्याला गॅट्सबीच्या [client-side](/docs/glossary#client-side) सह संवाद साधण्यासाठी बरेच पर्याय देते.
```
Comment (visshaljagtap) [inline docs/docs/api-files-gatsby-browser.md]: removed extra "[" in [[Gatsby ब्राउझर API]

File: docs/docs/api-files-gatsby-browser.md [hunk 2]
```diff
@@ -2,11 +2,11 @@
...
 
-The APIs `wrapPageElement` and `wrapRootElement` exist in both the browser and [Server-Side Rendering (SSR) APIs](/docs/ssr-apis). If you use one of them, consider if you should implement it in both `gatsby-ssr.js` and `gatsby-browser.js` so that pages generated through SSR with Node.js are the same after being [hydrated](/docs/glossary#hydration) with browser JavaScript.
+`wrapPageElement` आणि  `wrapRootElement` हे दोन्ही ब्राउझर मध्ये असित्वात आहेत  [Server-Side Rendering (SSR) APIs](/docs/ssr-apis).आपण त्यापैकी एखादा वापरत असल्यास, आपण दोन्ही `gatsby-ssr.js` आणि `gatsby-browser.js` जेणेकरुन SSR आणि Node.js केलेली पेज एकसारखीच असतील [hydrated](/docs/glossary#hydration) आणि ते ब्राउझर javascript मध्ये पण चालेल.
```
Comment (visshaljagtap) [inline docs/docs/api-files-gatsby-browser.md]: fix the typo असित्वात

Comment (visshaljagtap) [inline docs/docs/api-files-gatsby-browser.md]: आपण दोन्ही => samja aapn donhi mde implement kele pahije

Comment (visshaljagtap) [inline docs/docs/api-files-gatsby-browser.md]: जेणेकरुन SSR आणि Node.js केलेली पेज एकसारखीच असतील => जेणेकरुन SSR आणि Node.js ne generate केलेली पेज एकसारखीच असतील

**Review by visshaljagtap — COMMENTED** [review]
> ill take the smaller files and finish asap let's focus on the big files @visshaljagtap you take few files. I am also planning to take a few.

It will be better if we start finishing the pages under tutorial first! @spielers

**Comment by spielers** [comment]
@visshaljagtap lets merge this. I take part zero for now.

**Review by visshaljagtap — APPROVED** [review]

**Pull request merged**